netty/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
     // Tests depend on base class defined by core module.
     testCompile project(':grpc-core').sourceSets.test.output,
                 project(':grpc-testing')
+    signature "org.codehaus.mojo.signature:java17:+@signature"
```

Comment:
The Netty transport's TLS code doesn't actually work with JDK 6. There is a way to mark exclusions, but I didn't at this point.